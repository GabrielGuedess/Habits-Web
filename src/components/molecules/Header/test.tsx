import { render, screen } from '@testing-library/react';

import { Header } from '.';

jest.mock('components/atoms/Button', () => ({
  __esModule: true,
  Button: function Mock({ children }: { children: React.ReactNode }) {
    return <div data-testid="Button Mock">{children}</div>;
  },
}));

describe('<Header />', () => {
  it('should render the Header', () => {
    const { container } = render(<Header />);

    expect(screen.getByTestId('Button Mock')).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
