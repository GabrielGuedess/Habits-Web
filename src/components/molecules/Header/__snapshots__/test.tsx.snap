// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render the Header 1`] = `
<header
  class="mx-auto flex w-full max-w-3xl items-center justify-between"
>
  <img
    alt="Habits"
    data-nimg="1"
    decoding="async"
    height="71.49"
    src="/svg/logo.svg"
    style="color: transparent;"
    width="147.84"
  />
  <div
    data-testid="Button Mock"
  />
</header>
`;
